crates/rust-client/src/transaction/request/mod.rs

```diff
@@ -198,6 +201,16 @@ impl TransactionRequest {
         self.ignore_invalid_input_notes
     }
 
+    /// Returns the script arg for the transaction request.
+    pub fn script_arg(&self) -> &Option<Word> {
+        &self.script_arg
+    }
+
+    /// Returns the auth arg for the transaction request.
```

Comment:
I would maybe not abbreviate "argument" here (and in the above doc comment as well):

```suggestion
    /// Returns the auth argument for the transaction request.
```

I think we use "argument" when talking about `NoteArgs` in this same struct. I don't mind switching to "arg" but would be nice to do it throughout

Reply:
makes sense, done in [chore: dont abbreviate argument](https://github.com/0xMiden/miden-client/pull/1121/commits/10152e6f97d8bcc1b1e091c1407ba3dfa40590d5)